src/com_tjnotifications/admin/models/notifications.php

```diff
@@ -99,7 +99,10 @@ protected function getListQuery()
 		$orderCol  = $this->getState('list.ordering');
 		$orderDirn = $this->getState('list.direction');
 
-		$query->order($db->quoteName($orderCol) . ' ' . $db->escape($orderDirn));
+		if ($orderCol && $orderDirn)
```

Comment:
A lot of this code will already be present in the parent class. Can't we do `parent::getListQuery()` followed by only specific code ?

cc @vijaykhollam @manojLondhe @thite-amol @twsvaishali

Reply:
Don't think so, I think `parent::populateState()` does lot of stuff but `getListQuery` you need to write

Reply:
ok